### PR TITLE
Fix: fix partners name typo in tests

### PR DESCRIPTION
**Pull request by chyroc**

### CODE REVIEW

File: libs/partners/anthropic/tests/integration_tests/test_chat_models.py [hunk 1]
```diff
@@ -5,15 +5,15 @@
 
 
 def test_stream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from anthropic."""
```
Comment (rancomp) [inline libs/partners/anthropic/tests/integration_tests/test_chat_models.py:8]: I think they go with uppercase A;

```suggestion
    """Test streaming tokens from Anthropic."""
```

File: libs/partners/anthropic/tests/integration_tests/test_chat_models.py [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
     llm = ChatAnthropicMessages(model_name="claude-instant-1.2")
 
     for token in llm.stream("I'm Pickle Rick"):
         assert isinstance(token.content, str)
 
 
 async def test_astream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from anthropic."""
```
Comment (rancomp) [inline libs/partners/anthropic/tests/integration_tests/test_chat_models.py:16]: same here,
```suggestion
    """Test streaming tokens from Anthropic."""
```

File: libs/partners/nvidia-trt/tests/integration_tests/test_llms.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 @pytest.mark.skip(reason="Need a working Triton server")
 def test_stream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from nvidia-trt."""
```
Comment (rancomp) [inline libs/partners/nvidia-trt/tests/integration_tests/test_llms.py:11]: and they go all uppercase
```suggestion
    """Test streaming tokens from NVIDIA TRT."""
```

File: libs/partners/nvidia-trt/tests/integration_tests/test_llms.py [hunk 4]
```diff
@@ -17,7 +17,7 @@ def test_stream() -> None:
 
 @pytest.mark.skip(reason="Need a working Triton server")
 async def test_astream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from nvidia-trt."""
```
Comment (rancomp) [inline libs/partners/nvidia-trt/tests/integration_tests/test_llms.py:20]: likewise here,

and they go all uppercase
```suggestion
    """Test streaming tokens from NVIDIA TRT."""
```

**Review by rancomp — COMMENTED** [review]
I noticed a few fine details, but this PR is concerned with these exact fine details.

**Review by hwchase17 — APPROVED** [review]
thank you @chyroc and @rancomp both!!!

**Pull request merged**